allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
so it can be in separate place, but it usually will be in same place as long-line block